sklearn/ensemble/_forest.py

```diff
@@ -1436,9 +1427,18 @@ class RandomForestRegressor(ForestRegressor):
     .. [2] P. Geurts, D. Ernst., and L. Wehenkel, "Extremely randomized
            trees", Machine Learning, 63(1), 3-42, 2006.
 
-    See Also
+    Examples
     --------
-    DecisionTreeRegressor, ExtraTreesRegressor
+    >>> from sklearn.ensemble import RandomForestRegressor
+    >>> from sklearn.datasets import make_regression
+
```

Comment:
Could you remove this blank line? It will make 2 different blocks in the HTML rendering.